### PR TITLE
LRDOCS-4932, Forms and Workflow

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4932

### CODE REVIEW

File: discover/portal/articles/130-forms/10-workflow-and-forms.markdown [hunk 1]
```diff
@@ -1,80 +1,90 @@
...
-Approver workflow that ships with @product@ and you're the administrative user,
-you'll see a notification come in after submitting the form entry, alerting you
-that there's a form record to review. 
+Test the workflow process:
+
+1.  Add the form to a page.
+
+2.  Click *Submit to Publication* to submit the form entry.
+
+3.  Approve the entry.
```
Comment (natocesarrego) [inline discover/portal/articles/130-forms/10-workflow-and-forms.markdown]: It tells the reader to approve the form entry here and then ask it again in the next lines, therefore I think it's a little redundant. 😁

**Review by natocesarrego — COMMENTED** [review]
Just want to notify you that the screenshot and gif used in the last section (Example Use Case: Win a Free, All-Inclusive Trip to the Lunar Resort) are from version 7.0. 😉

Additionally, the pull is reviewed. 🙂

**Comment by rbohl** [comment]
@natocesarrego, we thought the GIF was similar enough between 7.0 and 7.1 that it didn't need to be updated. What do you think about it?

**Comment by natocesarrego** [comment]
Yeah, I agree they're very similar. My only concern is the scenario in which a reader is having his first contact with Forms, he never used Forms in 7.0. On this scenario, I believe the user can think it's an issue in the article, once the Form Builder UI is quite different in 7.1 due sidebar, for example. In addition, I believe the reader can become a little confused at a first seen when he tries to execute the same steps of the GIF in 7.1.

**Comment by natocesarrego** [comment]
Does this scenario make sense to you?

**Comment by rbohl** [comment]
Yep :wink:  comments addressed here: https://github.com/sez11a/liferay-docs/pull/3482

**Pull request merged**